src/Makefile.test_fuzz.include

```diff
@@ -17,5 +17,6 @@ libtest_fuzz_a_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES) $(BOOST_CPPFLAGS)
 libtest_fuzz_a_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
 libtest_fuzz_a_SOURCES = \
   test/fuzz/fuzz.cpp \
+  test/fuzz/mempool_utils.cpp \
```

Comment:
I wonder if we can nest more folders. Maybe `src/test/fuzz/util/mempool.cpp`?

Reply:
Added some nesting.